### PR TITLE
fix: live and department links

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Review by 120EE0692 — APPROVED** [review]

File: client/src/components/widgets/SubCategoryLink.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Link from 'next/link';
+
+// Libraries
+import { ButtonBase } from '@mui/material';
+
```
Comment (sambit-sankalp) [inline client/src/components/widgets/SubCategoryLink.js:6]: Remove these unnecessary lines.

File: client/src/components/widgets/SubCategoryLink.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+};
+
+export default SubCategoryLink;
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: 'inline',
+    textDecoration: 'none',
+  },
+
```
Comment (sambit-sankalp) [inline client/src/components/widgets/SubCategoryLink.js:30]: same here

File: client/src/components/widgets/SubCategoryLink.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    backgroundColor: ({ isRed }) =>
+      isRed ? 'rgba(215, 36, 46, 0.6)' : theme.palette.primary.blue10,
+    borderRadius: '4px',
+    margin: '8px 8px 0px 0px',
+    padding: '12px 16px',
+    [theme.breakpoints.down('md')]: {
+      padding: '3px 8px',
+    },
+  },
+
```
Comment (sambit-sankalp) [inline client/src/components/widgets/SubCategoryLink.js:42]: same here

**Review by sambit-sankalp — CHANGES_REQUESTED** [review]
Make sure to run prettier before committing your code.

**Review by sambit-sankalp — APPROVED** [review]
GTG

File: client/src/pages/live.jsx [hunk 4]
```diff
@@ -74,6 +74,15 @@ const LivePage = () => {
...
 
+export async function getServerSideProps() {
+  return {
+    redirect: {
+      destination: '/comingSoon',
+      permanent: false,
+    },
+  };
+}
+
```
Comment (rutajdash) [inline client/src/pages/live.jsx:85]: Why is this code here?

Comment (Shurtu-gal) [reply client/src/pages/live.jsx]: Due to merging of PR #391 .

It has been removed now.

**Pull request merged**